collect-user-data

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
```

Comment:
My convention at this point is do it in bash if it's not too complicated.  Typically it gets complicated when you have to create or manipulate complex data structures.  

This collect-user-data script is probably a one-off (might not be used again) and not very future proof/future friendly.  I would consider removing it completely and taking the useful bits (interesting queries) and putting them in the cookbook.  I ended up using a different method to get a list of docs related to a user anyway (replicate and then get a UUID list)... because when I replicated and when I queried views I got different results.